uwrt_mars_rover_blackfly_calibration/uwrt_mars_rover_blackfly_calibration/broadcaster.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
```

Comment:
so this node publishes images from the blackfly. in hindsight, i dont think this node should exist here. instead, this package should just be running the actual blackfly node (the node we'd run at mission time). should i create a different package for the blackflys?

Reply:
Isn't there and actual flir node already?

Reply:
:O where?

Reply:
There's this somewhat abandoned "official" one from clearpath: https://github.com/ros-drivers/flir_camera_driver but i dont see ros2 support. This is what we used to use i think.

I just found this one: https://github.com/berndpfrommer/flir_spinnaker_ros2